### DISCUSSION TITLE
ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The media could not be loaded, either because the server or network failed or because the format is not supported.

### DISCUSSION THREAD

**Discussion by gilmotta** [videojs/video.js#7729] [Q&A]
It looks like that #4751 was closed without a fix. We are getting the same error code. Our connection is secure over HTTPS and it works on Firefox and Chrome but playing thru the Android App webview we get the error.

Can anyone please let us know if there is a fix or workaround to use video.js through the webview to play DRM content?

Thanks,

**Comment by mister-ben** [comment]
This is a generic error coming from the video element that tells you the video can not be played, but there can be many causes, most of which are out of Video.js's control. That issue didn't have an example that could be reproduced, and then others added comments because they's seen the same error as a symptom of almost certainly different causes.

A webview doesn't have all the capabilities of a proper browser, but some can be enabled with some additional configuration. For DRM you need to handle a permissions request as the [comment here](https://github.com/videojs/video.js/issues/5563#issuecomment-437327828).
